lib/Workflow.pm

```diff
@@ -377,6 +378,30 @@ sub _get_next_state {
     return $wf_state->get_next_state( $action_name, $action_return );
 }
 
+sub _auto_execute_state {
```

Comment:
I'll submit a follow-up PR: This function was modified on the branch you merged in, but removed on the PR. The modified version is - just as the original - irrelevant after the changes in the PR. I'll follow up with a PR to remove this function.